version.js

```diff
@@ -0,0 +1,3 @@
+// TODO: AUTO GENERATED FILE, DO NOT EDIT
```

Comment:
is this a TODO or a warning? this file does not need to be auto generated to start

Reply:
Its a todo, clarified it.